handler.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const request = require('superagent')
+const async = require('async')
+const config = require(`./config/production.json`)
```

Comment:
It is better to have a dummy/sample `production.json` file instead of letting it failed. It makes first time user easier to start. However, a `README` file that explain this will be better.